### PR TITLE
Update protocol clients list

### PR DESCRIPTION
**Pull request by gitbuda**

### CODE REVIEW

File: products/ha_memgraph/v0.1.0/how_to_guides/query-ha-memgraph-programmatically.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ for some popular programming languages are listed below:
   * [JavaScript](https://github.com/neo4j/neo4j-javascript-driver)
   * [C#](https://github.com/neo4j/neo4j-dotnet-driver)
   * [Ruby](https://github.com/neo4jrb/neo4j)
+  * [Rust](https://github.com/memgraph/rsmgclient)
   * [Haskell](https://github.com/zmactep/hasbolt)
   * [PHP](https://github.com/graphaware/neo4j-bolt-php)
```
[the remark this answers is not in the document]
Comment (gitbuda) [reply products/ha_memgraph/v0.1.0/how_to_guides/query-ha-memgraph-programmatically.md]: I agree + used https://www.textfixer.com/tools/alphabetical-order.php (C# vs C/C++ was questionable from my perspective).

**Review by jbajic — APPROVED** [review]

**Comment by gitbuda** [comment]
I would remove the examples because it is more-less copy-pasted from documentations for each specific client. Except, JavaScript example at the end because that has additional info?

**Comment by jbajic** [comment]
> I would remove the examples because it is more-less copy-pasted from documentations for each specific client. Except, JavaScript example at the end because that has additional info?

Java example also seems useful, but I agree.

**Comment by gitbuda** [comment]
I'll merge as-is. We'll make restructuring in some future PR.

**Pull request merged**